index.html

```diff
@@ -30,7 +30,13 @@ <h1>Welcome to the student showcase!</h1>
         <div>TEN</div>
         <div>ELEVEN</div>
         <div>TWELVE</div>
-        <div>THIRTEEN</div>
+        <div>
+            <h2>Jason Brown</h2>
+            <img src="https://pbs.twimg.com/profile_images/1341286622/speedo_gnome_400x400.jpg" alt="My Photo">
+            <a href="https://twitter.com/theyardgnome">@TheYardGnome</a></li>
+            <h3>I love Lamp!</h3>
+        
```

Comment:
Please remove this extra space!

Reply:
Are you talking about where my div closes?

Reply:
I'm talking about removing line 38. Notice how there's now an additional line with nothing on it!